C#/number-sort.cs

```diff
@@ -2,7 +2,7 @@ public class Sort
 {
```

Comment:
You must also import `System` or else it breaks the program, and Array and Console remain unidentified.